plugins/techdocs-backend/src/default-branch.ts

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Would be nice with tests for this file. We can add them after if you're eager to get this merged though. :)

Reply:
Yes, I would be happy if you add tests after merge)